cli/tests/integrations/test_package.py

```diff
@@ -84,7 +84,7 @@ def test_repo_list():
         assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
 
 
-def test_repo_add():
+def test_repo_add_and_remove():
```

Comment:
Why do you combine these now?

Reply:
they were order-dependent, cf. the commit message.